consul/src/main/scala/io/buoyant/consul/utils.scala

```diff
@@ -0,0 +1,63 @@
+package io.buoyant.consul
+
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.{Failure, Http, Service, SimpleFilter}
+import com.twitter.util.Monitor
+import io.buoyant.config.types.Port
+import java.net.InetSocketAddress
+
+
+object utils {
+
+  implicit class RichConsulClient(val client: Http.Client) {
```

Comment:
OK. I'd be very grateful if you could briefly explain what it gives us or point me to some doc about that.